packages/nimble/inst/tests/test_utils.R

```diff
@@ -292,16 +292,16 @@ test_coreRfeature_internal <- function(input, verbose = TRUE, dirName = NULL) {
   invisible(NULL)
 }
 
-gen_runFun <- function(input) {
+gen_runFun <- function(param) {
```

Comment:
I've renamed the argument from generic `input` to specific `param` to match standard terminology of "parametrized tests".